python/cudf/cudf/core/column/categorical.py

```diff
@@ -1214,6 +1214,18 @@ def view(self, dtype):
             "Categorical column views are not currently supported"
         )
 
+    def sum(self, dtype=None):
+        raise TypeError("Categorical cannot perform the operation sum")
+
+    def product(self, dtype=None):
+        raise TypeError("Categorical cannot perform the operation prod")
+
+    def std(self, ddof=1, dtype=np.float64):
+        raise TypeError("Categorical cannot perform the operation std")
+
+    def var(self, ddof=1, dtype=np.float64):
+        raise TypeError("Categorical cannot perform the operation var")
```

Comment:
Instead of defining these here should these just be in the `ColumnBase` class and then Numerical can override them?

Reply:
Pushed the apis to `Column`